service/src/main/java/org/cbioportal/service/impl/CopyNumberEnrichmentServiceImpl.java

```diff
@@ -29,39 +29,45 @@
     private MolecularProfileService molecularProfileService;
     @Autowired
     private AlterationEnrichmentUtil alterationEnrichmentUtil;
-    
+
     @Override
-    public List<AlterationEnrichment> getCopyNumberEnrichments(String molecularProfileId, List<String> alteredIds,
-                                                               List<String> unalteredIds, List<Integer> alterationTypes, 
-                                                               String enrichmentType)
-        throws MolecularProfileNotFoundException {
+    public List<AlterationEnrichment> getCopyNumberEnrichments(List<Entity> set1, List<Entity> set2, List<Integer> alterationTypes, String enrichmentType) throws MolecularProfileNotFoundException {
```

Comment:
Please avoid having too many parameters in a single line, it reduces the code readability. Go along with the existing style, unless you have a good reason not to.